src/views/splash/splash.jsx

```diff
@@ -106,12 +114,33 @@ var Splash = injectIntl(React.createClass({
             if (!err) return this.setState({featuredGlobal: body});
         }.bind(this));
     },
-    getFeaturedCustom: function () {
+    getSharedByFollowing: function () {
         api({
-            uri: '/proxy/users/' + this.props.session.session.user.id + '/featured'
+            uri: '/projects/following/users',
+            authentication: this.props.session.session.user.token
```

Comment:
You probably didn't get the information you were looking for because the tokens you are referring to are not the ones that will be used for authentication of this endpoint – rather, the token will eventually be retrieved from your session, though that change is not yet on the website.

Also, api-staging is an internal endpoint that we use for testing and is not reflective of the information that is currently on Scratch.